.github/workflows/main.yml

```diff
@@ -0,0 +1,31 @@
+name: unzip-http-ci-build
+on:
+  pull_request:
+    branches:
+    - master
+  push:
+    branches:
+    - master
+
+jobs:
+
+  run-tests:
+
+    strategy:
+      matrix:
+        python-version: [3.8, 3.9, 3.10]
+
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+
+    - name: Install unzip-http
+      run: |
+        python setup.py develop
+    - name: Run basic test
+      run: unzip-http https://geonames.nga.mil/gns/html/cntyfile/geonames_20220606.zip "*.txt"
```

Comment:
Let's find a test that only uses a few kb :)

Reply:
`unzip-http http://psa.download.navigation.com/automotive/PSA/RT6-SMEGx/M49RG20-Q0420-2001.ZIP DATA/CURR_VERS_NAVI.TXT`